fast64_internal/f3d/f3d_material.py

```diff
@@ -589,14 +589,15 @@ def ui_chroma(self, material, layout, name, setName, setProp, showCheckBox):
         inputGroup = layout.row()
         prop_input_name = inputGroup.column()
         prop_input = inputGroup.column()
+        f3d_mat = material.f3d_mat
```

Comment:
You accidentally removed a new line here. ui_env and ui_image have a new line like that but prim and light don't so its fine really.